web/frontend/src/app/component/evs-header/evs-header.component.ts

```diff
@@ -132,4 +133,5 @@ export class EvsHeaderComponent implements OnInit {
     this.truncated = !this.truncated;
   }
 
+  protected readonly str = str;
 }
```

Comment:
You're importing "str" from "ajv" and I can't see what this is for and seems like we should remove this if its not doing something we need.

Reply:
Weird, I didn't add that. But it's not doing anything, so I'll remove it